cmd/auth-provider-gcp/plugin/plugin.go

```diff
@@ -0,0 +1,41 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package plugin
+
+import (
+	"fmt"
+)
+
+// TODO(DangerOnTheRanger): temporary structure until credentialprovider
+// is built with cloud-provider-gcp; GetAuthPluginResponse should return
+// CRIAuthPluginResponse instead, but this should be nearly a drop-in replacement
+type pluginResponse struct {
```

Comment:
plugin seems confusing here. In the K/K code there were several "plugins" and you picked which one you wanted. This binary essentially is the equivalent to the old plugin. This is either the response or a credentialResponse.

Reply:
Done